src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './TodoList.scss';
+import { PreparedTodo } from '../../types/PreparedTodo';
+import { UserInfo } from '../UserInfo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  preparedTodos: PreparedTodo[]
+};
+
+export const TodoList: React.FC<Props> = ({ preparedTodos }) => (
+  <table>
+    <thead>
+      <td>email</td>
+      <td>name</td>
+      <td>title</td>
+      <td>completed</td>
+    </thead>
+    {preparedTodos.map(todo => (
+      <tr>
```

Comment:
Атрибут key?

Reply:
ок

Reply:
Ты опять просто переименовал название ключа
```suggestion
      <tr key={todo.id}>
```

Reply:
уга, уже догнал с совета в другой задачи))